### PR TITLE
Fix/update fixed code

### PR DESCRIPTION
**Pull request by BinLiang-Eric**
- A crash that occurred when enabling accessibility before starting a study has been fixed.
- The screen text sensor now correctly collects duplicate text when the foreground app changes, ensuring that screen text is captured during app switches, home screen access, and notification checks.
- Fix display issues: long question titles can be displayed completely.
- Fix the ESM queue logic to ensure that the correct number of questions are displayed according to the configurator settings.
- Fix logic of reschedule next day random esm trigger, avoid repeat appear of esm notification.

### CODE REVIEW

File: aware-core/aware.gradle
Comment (tinazhang128) [inline aware-core/aware.gradle:1]: Why this was originally 3 rather than 5. I thought this is 4.1.5.light.bundle

File: CHANGES.txt
Comment (tinazhang128) [inline CHANGES.txt:1]: It seems like one more change has been added to the code: a stronger password detection added. Could you please add this into the CHANGES.TXT

File: aware-core/aware.gradle
Comment (BinLiang-Eric) [reply aware-core/aware.gradle:1]: The version code in aware-core is indeed 5. I think [this commit](https://github.com/awareframework/aware-light-client/commit/4e69013285b73ff014a19873e4c1b429115053ff#diff-59130575b4fb2932c957db2922977d7d89afb0b2085357db1a14615a2fcad776L20) caused a mismatch in the version number for aware-phone.

**Review by tinazhang128 — APPROVED** [review]
LGTM!

**Pull request merged**